contracts/LinniaRecords.sol

```diff
@@ -9,28 +9,39 @@ import "./LinniaUsers.sol";
 contract LinniaRecords is Ownable {
     using SafeMath for uint;
 
-    struct FileRecord {
-        address fileOwner;
-        uint sigCount;
+    // Struct of a linnia record
+    // A linnia record is identified by its root hash, which is
+    // sha(sha(data), sha(metadata))
+    struct Record {
+        // owner of the record
+        address owner;
+        // hash of the data before its encrypted
+        bytes32 dataHash;
+        // hash of the plaintext metadata
+        bytes32 metaHash;
+        // attestator signatures
         mapping (address => bool) sigs;
+        // count of attestator sigs
```

Comment:
This isn't true. According to [this](https://github.com/ConsenSys/Linnia-Smart-Contracts/blob/fix/metamalleability/contracts/LinniaRecords.sol#L220), it's the sum of provenance scores of attestators. I would say that both the name and the comment are wrong // OR (highly likely), the code is wrong and it's `record.sigCount += 1` (instead of `record.sigCount = record.sigCount.add(provenanceScore);`). Right now `sigCount` and `irisScore` are the same thing. `sigCount`->`sigsCount` would be a better term. It could also be [something more standard out of the typical patterns](https://ethereum.stackexchange.com/a/13168).